src/runtime_src/core/common/memaccess.cpp

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (C) 2016-2021 Xilinx, Inc
+ * Copyright (C) 2022 Advanced Micro Devices, Inc. - All rights reserved
+ * 
+ * Author: Sonal Santan
+ * Simple command line utility to inetract with SDX PCIe devices
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+#define XRT_CORE_COMMON_SOURCE
+#include "memaccess.h"
+
+#include <iostream>
+#include <numeric>
+#include <vector>
+
+#include "memalign.h"
+#include "query_requests.h"
+#include "utils.h"
+#include "unistd.h"
```

Comment:
`<unistd.h>`?

Reply:
This is okay as is.

Reply:
Argh! Could it be renamed something else just to be clearer?

Reply:
Fair enough.  Confusion isn't good.
But let's instead be specific about the include as it was before as @dbenusov-xilinx suggested.